src/components/article-card/article-card.jsx

```diff
@@ -1,17 +1,30 @@
-function ArticleCard({
-  id,
-  image,
-  author,
-  commentCount,
-  date,
-  title,
-  topic,
-  votes,
-}) {
+import { useNavigate } from "react-router-dom";
+import "./article-card.css";
+
+function ArticleCard(prams) {
+  const { id, image, author, date, title, topic } = prams;
+  const navigate = useNavigate();
+
+  const handleCardClick = () => {
+    navigate(`/articles/${id}`);
```

Comment:
This should be a Link component. 
Using Link renders a standard `a href` tag in the html which tells a screen reader that it is a link. This is for accessibility.
Navigate is used for programmatic navigation, when you want to send the user somewhere. For example, if I log in on a website it will probably take me somewhere after I fill in my details, that would be using Navigate. When the user wants to go somewhere that should be a Link tag.